elements/pfe-sass/variables/_maps.scss

```diff
@@ -161,13 +161,15 @@ $pfe-vars: (
 
   font-size:                     16px,
   line-height:                   1.5,
-  line-height--heading:          1.3,
-  line-height--heading-xl:       1.2,
+  line-height--heading:          1.2,
+  line-height--heading--alpha:   1.3,
```

Comment:
Not sure about this naming, maybe we keep it as xl, I was just trying to sort it out first in the code.